e2e/plugins/index.js

```diff
@@ -1,6 +1,35 @@
+/* eslint-disable */
 const { initPlugin } = require("cypress-plugin-snapshots/plugin");
+const execa = require("execa");
 
 module.exports = (on, config) => {
+  on("task", {
+    installPlugin({ name }) {
+      // Because cypress commands are async but are not promises,
+      // we have to wrap all task functions with a new Promise constructor.
+      // https://stackoverflow.com/a/49980928/2638310
+      //
+      // Asynchronous tasks HAVE TO return a promise.
+      // https://docs.cypress.io/api/commands/task.html#Return-a-Promise-from-an-asynchronous-task
+      //
+      //
+      return new Promise((resolve, reject) => {
```

Comment:
Async functions do return a Promise themselves so I think it should be enough if you'd return the async function directly 🙂